infra/ursa/init-letsencrypt.sh

```diff
@@ -27,33 +24,31 @@ if [ ! -e "$data_path/conf/options-ssl-nginx.conf" ] || [ ! -e "$data_path/conf/
   echo
 fi
 
-echo "### Creating dummy certificate for $domains ..."
```

Comment:
why we capitalizing variable

Reply:
its changed. But the variable now provided to the script is capitalized